include/mqtt/message.hpp

```diff
@@ -187,9 +187,8 @@ struct basic_pubrel_message : detail_v3_1_1::basic_header_packet_id_message<Pack
     {
     }
 
-    template <typename Iterator>
-    basic_pubrel_message(Iterator b, Iterator e)
-        : base(b, e)
+    basic_pubrel_message(buffer buf)
```

Comment:
const&

Reply:
Ah, yes MQTT v3.1.1's pubrel message is fixed length message. I will fix it.

Reply:
I changed as follows for consistency.

```cpp
basic_pubrel_message(mqtt::string_view)
```